server/views/applications/pages/basic-information/release-date.njk

```diff
@@ -22,16 +22,33 @@
       fetchContext()
     )
   }}
+  {% set html %}
+  <p>
+    Your application will be sent to the central referral unit(CRU) to match the person to a suitable AP.You will receive an email
```

Comment:
there should be a space between `unit` and `(CRU)` and one between the full stop and `You`